src/qt/bitcoingui.cpp

```diff
@@ -1084,6 +1092,30 @@ void BitcoinGUI::setEncryptionStatus(int status)
 }
 #endif // ENABLE_WALLET
 
+void BitcoinGUI::updateProxyIcon()
+{
+    bool proxyEnabled = false;
+    proxyType proxy;
+    UniValue networks(UniValue::VARR);
+
+    for(int n=0; n<NET_MAX; ++n)
+    {
+        enum Network network = static_cast<enum Network>(n);
+        if(network == NET_UNROUTABLE || network == NET_INTERNAL)
+            continue;
+        GetProxy(network, proxy);
+        if(proxy.IsValid()) {
+            proxyEnabled = true;
+            break;
+        }
+    }
+
+    labelProxyIcon->show();
+    labelProxyIcon->setPixmap(platformStyle->SingleColorIcon(proxyEnabled ? ":/icons/proxy_enabled" : ":/icons/proxy_disabled").pixmap(STATUSBAR_ICONSIZE,STATUSBAR_ICONSIZE));
```

Comment:
Space after `,`.

Nit, split in 2 lines? Or something like:

```
if (proxy_enabled) {
   ...
} else {
   ...
}
```

Reply:
Removed space, prefer it like this for now.